src/gpio.rs

```diff
@@ -1076,6 +1082,40 @@ mod chip {
     pin!(Gpio38:38, Input, RTC:2, ADC1:2, NODAC:0, NOTOUCH:0);
     pin!(Gpio39:39, Input, RTC:3, ADC1:3, NODAC:0, NOTOUCH:0);
 
+    impl<MODE> RmiiClkPin for Gpio0<MODE>
+    where
+        MODE: Send,
```

Comment:
Nit: I don't think you need the `Send` constraint here?

Reply:
I copied it, so no :)

Reply:
actually, it is needed for pin to be send. I just tried without